### PR TITLE
Update changelog for September 19 2024

### PR DESCRIPTION
**Pull request by iroussos**
# Description

Update changelog for September 19 2024

* New modes in Timescale Console: Ops and Data mode
* highlight a statement to run on SQL editor

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/changelog.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+**To better serve you, we've introduced new modes to the Timescale Console UI—tailoring the experience based on what you're trying to accomplish.**
+
+Ops mode is where you can manage your services, add replicas, configure compression, change parameters, etc. 
+
+Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.
+
+Try it today and let us know what you think! 
+
+![Timescale Console Ops and Data mode]( !!! URL !!! )
```
Comment (iroussos) [inline about/changelog.md]:
```suggestion
![Timescale Console Ops and Data mode](https://s3.amazonaws.com/assets.timescale.com/docs/images/ops-data-mode.gif)
```

**Comment by iroussos** [comment]
Looks good on my side - feel free to make any changes you want and merge without waiting for me to be available tomorrow my morning :-)

**Review by rahilsondhi — APPROVED** [review]

**Review by billy-the-fish — APPROVED** [review]
LGTM

File: about/changelog.md [hunk 2]
```diff
@@ -8,6 +8,33 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.
+
+Try it today and let us know what you think! 
+
+![Timescale Console Ops and Data mode](https://assets.timescale.com/docs/images/ops-data-mode.gif)
+
+## Console based Parquet File Import
+
+* Now users can upload their Parquet files to Timescale by simply selecting the file from their local file system.
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
* Now users can upload from Parquet  to Timescale by uploading the file from their local file system.  For files larger than 250 MB or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.
```

File: about/changelog.md [hunk 3]
```diff
@@ -8,6 +8,33 @@ keywords: [changelog, upgrades, updates, releases]
...
+Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.
+
+Try it today and let us know what you think! 
+
+![Timescale Console Ops and Data mode](https://assets.timescale.com/docs/images/ops-data-mode.gif)
+
+## Console based Parquet File Import
+
+* Now users can upload their Parquet files to Timescale by simply selecting the file from their local file system. 
+* For files larger than 250 MB or if users prefer to do it themselves, they can follow a three-step process to upload Parquet files into Timescale.
```
Comment (billy-the-fish) [inline about/changelog.md]: Delete please

File: about/changelog.md [hunk 4]
```diff
@@ -8,6 +8,33 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+https://github.com/user-attachments/assets/b1048bfe-e2a2-433a-a767-d7e00a20f715
+
+### Other improvements
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
## SQL editor improvements
```

File: about/changelog.md [hunk 5]
```diff
@@ -8,6 +8,33 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+* In the Ops mode SQL editor, you can now highlight a statement to just run that specific statement
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
* In the Ops mode SQL editor, you can now highlight a statement to run a specific statement.
```

File: about/changelog.md [hunk 6]
```diff
@@ -8,6 +8,32 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+Data mode is the full PopSQL experience: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.
+
+Try it today and let us know what you think! 
+
+![Timescale Console Ops and Data mode](https://assets.timescale.com/docs/images/ops-data-mode.gif)
+
+## Console based Parquet File Import
+
+* Now users can upload from Parquet  to Timescale by uploading the file from their local file system.  For files larger than 250 MB or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
Now users can upload from Parquet to Timescale Cloud by uploading the file from their local file system. For files larger than 250 MB, or if you want to do it yourself, follow the three-step process to upload Parquet files to Timescale.
```

File: about/changelog.md [hunk 7]
```diff
@@ -8,6 +8,32 @@ keywords: [changelog, upgrades, updates, releases]
...
+
+https://github.com/user-attachments/assets/b1048bfe-e2a2-433a-a767-d7e00a20f715
```
Comment (billy-the-fish) [inline about/changelog.md]:
```suggestion
![Upload from Parquet to Timescale Cloud](https://assets.timescale.com/docs/images/upload-parquet.gif)
```

**Pull request merged**